arches_lingo/src/arches_lingo/components/concept/ConceptMatch/ConceptMatch.vue

```diff
@@ -0,0 +1,130 @@
+<script setup lang="ts">
+import { onMounted, ref } from "vue";
+
+import Message from "primevue/message";
+import ProgressSpinner from "primevue/progressspinner";
+
+import ConceptMatchEditor from "@/arches_lingo/components/concept/ConceptMatch/components/ConceptMatchEditor.vue";
+import ConceptMatchViewer from "@/arches_lingo/components/concept/ConceptMatch/components/ConceptMatchViewer.vue";
+
+import { EDIT, VIEW } from "@/arches_lingo/constants.ts";
+
+import { fetchLingoResourcePartial } from "@/arches_lingo/api.ts";
+
+import type {
+    ConceptMatchStatus,
+    DataComponentMode,
+} from "@/arches_lingo/types.ts";
+
+const props = defineProps<{
+    mode: DataComponentMode;
+    sectionTitle: string;
+    componentName: string;
+    graphSlug: string;
+    nodegroupAlias: string;
+    resourceInstanceId: string | undefined;
+    tileId?: string;
+}>();
+
+const isLoading = ref(true);
+const tileData = ref<ConceptMatchStatus[]>([]);
+const schemeId = ref<string>();
+const fetchError = ref();
+
+const shouldCreateNewTile = Boolean(props.mode === EDIT && !props.tileId);
+
+onMounted(async () => {
+    if (
+        props.resourceInstanceId &&
+        (props.mode === VIEW || !shouldCreateNewTile)
+    ) {
+        const sectionValue = await getSectionValue();
+        tileData.value = sectionValue.aliased_data[props.nodegroupAlias];
+        schemeId.value = await getSchemeId();
+    }
+    isLoading.value = false;
+});
+
+async function getSectionValue() {
+    try {
+        const sectionValue = await fetchLingoResourcePartial(
+            props.graphSlug,
+            props.resourceInstanceId as string,
+            props.nodegroupAlias,
+        );
+        for (const value of sectionValue.aliased_data[props.nodegroupAlias]) {
+            value.aliased_data["uri"] = await getURI(
+                value.aliased_data["match_status_ascribed_comparate"][0][
+                    "resourceId"
+                ],
+            );
```

Comment:
this has the ability to spam the server. Could this whole `getSectionValue` be rewritten to only make a single call to the backend? you'd need a custom API